charts/hub-agent/values.yaml

```diff
@@ -33,3 +33,6 @@ affinity: {}
 
 secret:
   name: hub-kubeconfig-secret
+
+enableV1Alpha1APIs: true
+enableV1Beta1APIs: false
```

Comment:
Those are the default already, right? Do we still need this?

Reply:
Hi Ryan! This toggle is used for the (new) E2E tests, which installs agents using these helm charts and needs the v1beta1 APIs to be enabled.

Reply:
I merged too soon. Why is the enableV1Beta1API false then?